### ISSUE TITLE
any news on the update regarding the update for flutter 2.x ?

### ISSUE THREAD

**Issue by robertnax** [EPNW/dumble#8]
waiting patiently :)

**Comment by robertnax** [comment]
you guys stopped working on this ?

**Comment by EP-u-NW** [comment]
No, but its not clear to me what your question is 😅  Dumble already works with null safe dart and thus with current flutter versions. Some things need to be refined in the future (as listed in `ROADMAP.md`), but thats nothing that would prevent you from using it right now. In fact, we are already using it in some published flutter apps.
